### PR TITLE
Ui fixes

### LINKED ISSUE #35: As a user I can see the seller and price of properties

### LINKED ISSUE #37: Bug with large prices
Prices above 10^18 results in an error.

### PR DESCRIPTION
**Pull request by waynetee**
Closes #35, closes #37 
image
Will merge with one approving review

### CODE REVIEW

**Review by guangxuan — APPROVED** [review]
image
Lgtm
UI behaves as expected when strings are very long.

File: src/main/java/seedu/address/ui/PropertyCard.java [hunk 1]
```diff
@@ -40,6 +40,10 @@
     private Label email;
     @FXML
     private FlowPane tags;
+    @FXML
+    private Label price;
+    @FXML
+    private Label seller;
 
     /**
      * Creates a {@code PropertyCode} with the given {@code Property} and index to display.
```
Comment (lizchow) [inline src/main/java/seedu/address/ui/PropertyCard.java:49]:
```suggestion
     * Creates a {@code PropertyCard} with the given {@code Property} and index to display.
```

**Review by lizchow — APPROVED** [review]
LGTM. Just minor documentation nitpick thats not really your fault.

**Comment by waynetee** [comment]
Thanks @guangxuan for posting the screenshot and @lizchow for the good catch (very observant)

**Pull request merged**